tests/shared.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 
 
-def hex_code_check(tmp_path: Path, filename: str, ext: str, hex_code: str = "", return_hex=False):
+def hex_code_check(tmp_path: Path, filename: str, ext: str, hex_code: str = "", return_hex=False, does_not_equal = False):
```

Comment:
Not wrong but always try and simplify, if no standard exists. `does_not_equal=False` is the same as simpler `equals=True`.
And then you just reorder the if.